### DISCUSSION TITLE
Official email account for Rowy login

### DISCUSSION THREAD

**Discussion by msamsonraj** [AntlerVC/firetable#1432] [Support / Q&A] [answered]
Currently Rowy supports only Google, Github and Microsoft account. I want to use my official company email account to log-in.  Is it possible?

**Comment by Chadha93** [comment, marked as answer]
Hi @msamsonraj, Yes there are some workarounds to make this possible, few questions I may ask to help you.

Method 1. Does your official company email have access to Google Workspace? Or allows you to create one? If yes, yes you can simply create a Google account with it and sign up to Rowy using Google.

Method 2. Log in using your personal Google/GitHub/Microsoft account and invite your official company email account as a team member give a role as admin and transfer the whole workspace to your official company email account. Please refer to these docs to invite a user and transfer a workspace.
2.1 - https://docs.rowy.io/how-to/roles.
2.2 - https://docs.rowy.io/faqs/delete#leave-workspace.

Please let me know if this helps. Also, you can connect with the team using Discord - https://discord.gg/9UrnWysm

**Reply by Chadha93** [reply]
I'll also tag @BeeBombshell to confirm if there are any other workarounds or self-signup options to be available in the near future.
